run_scans.py

```diff
@@ -3,26 +3,25 @@
 def launch(args):
   return launchArgs(*args)
 
-def launchArgs(tool, shelf, slot, link, chamber, vfatmask, scanmin, scanmax, nevts, stepSize=1,
+def launchArgs(tool, cardName, shelf, link, chamber, vfatmask, scanmin, scanmax, nevts, startTime, stepSize=1,
                vt1=None,vt2=0,mspl=None,perchannel=False,trkdata=False,ztrim=4.0,
                config=False,amc13local=False,t3trig=False, randoms=0, throttle=0,
-               internal=False, debug=False):
-  import datetime,os,sys
+               internal=False, debug=False, voltageStepPulse=False):
+  import os,sys
   import subprocess
   from subprocess import CalledProcessError
   from gempython.gemplotting.mapping.chamberInfo import chamber_config
   from gempython.utils.wrappers import runCommand
 
-  startTime = datetime.datetime.now().strftime("%Y.%m.%d.%H.%M")
   dataPath = os.getenv('DATA_PATH')
-
+ # print 'dataPath = ', dataPath
```

Comment:
it is discouraged to include commented lines in the central repo as this impacts readability.

Remove.

Reply:
noted, my b